Makefile

```diff
@@ -36,7 +36,8 @@ build-local:
 	@mkdir -p "bin"
 	go build -tags experimental -o "bin/dnet" ./cmd/dnet
 	go build -o "bin/docker-proxy" ./cmd/proxy
-
+	go build -o "bin/cniserver" ./cmd/cni_server
+	go build -o "bin/cnictl" ./cmd/dnet_cni/
```

Comment:
I think this needs to be statically built since you don't control the environment it will end up running in -- it is run in the context of the runtime (so maybe cri or docker, perhaps in a container) and not in a container/env supplied by libnetwork.